ansible/pbTestScripts/testScript.sh

```diff
@@ -144,6 +144,15 @@ testBuild()
 	vagrant ssh -c "cd /vagrant/pbTestScripts && ./buildJDK.sh"
 }
 
+testBuildWin()
+{
+	# Ensures the git config won't change line endings
+	vagrant powershell -c "Start-Process powershell -Verb runAs; C:/cygwin64/bin/sed -i -e 's/autocrlf.*/autocrlf = false/g' C:\\ProgramData/Git/config"
+	vagrant powershell -c "cd C:/; if (-not (Test-Path C:/openjdk-build -PathType Container) ) { echo 'Cloning openJDK-build repo' ; git clone https://github.com/adoptopenjdk/openjdk-build ; sleep 3 }"
+	# Runs the build script via ansible
```

Comment:
Can you add a comment into this script saying why we're not using `vagrant powershell`to make it clear why we're doing something that superficially seems a bit odd ... Possibly include a link to https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/942#issuecomment-539946564